reach

```diff
@@ -115,10 +115,143 @@ run_s () {
   sh -ac "$TMP/out.sh" "$0"
 }
 
+gen_reachc_id () {
+  REACH_ENV_DIR="${XDG_CONFIG_HOME:-$HOME/.config}/reach/"
+  REACH_ENV_FILE="${REACH_ENV_DIR}/env"
+  MD5_CMD=md5sum||md5
+  PRE_REACHC_ID=$(head -n1 /dev/urandom | $MD5_CMD | head -c 30 | tr [a-z] [A-Z])
+
+  mkdir -p $REACH_ENV_DIR
+
+  if [ ! -e $REACH_ENV_FILE ]; then
+    echo "export REACHC_ID=${PRE_REACHC_ID}" > $REACH_ENV_FILE
+  elif ( ! grep REACHC_ID $REACH_ENV_FILE > /dev/null 2>&1 ); then
+    echo "export REACHC_ID=${PRE_REACHC_ID}" >> $REACH_ENV_FILE
+  fi
+  . $REACH_ENV_FILE
+}
+
+call_home () {
+  START_TIME=$1
+  RESULT=$2
+  REACH_VERSION=$3
+  CONNECTOR_MODE=$4
+  USING_VSSTUDIO_EXTENTION=$5
```

Comment:
"EXTENTION" is the typo we were talking about.

Reply:
English not been the 1st language error, haha.

Thanks for flagging this out.